packages/circuit-ui/components/ButtonGroup/ButtonGroup.tsx

```diff
@@ -109,10 +109,13 @@ const actionWrapperStyles = ({ theme }: StyleProps) => css`
 
   ${theme.mq.kilo} {
     flex-direction: row;
-    justify-content: center;
   }
 `;
 
+const actionAlignmentStyles = ({ align = 'center' }: ButtonGroupProps) => css`
+  justify-content: ${alignmentMap[align]};
+`;
```

Comment:
The alignment styles don't have any effect on mobile, is that intentional?

Reply:
Yes, since the `ButtonGroup` children change flex-direction (to `row`) on mobile, they're centered.

In edge cases such as a UI where we want buttons to remain inline on mobile (previously supported via the `inlineMobile` prop) we'll recommend writing a custom wrapper. This is reflected in the "Swiper" story updates.

How does this sound to you @connor-baer?

Reply:
Sounds good, thanks for clarifying! We'll have to review each use case in the dashboard then.

Reply:
Yes, for some we'll need to switch to a custom implementation. But I most implementations should probably follow component guidelines and wrap on mobile—the "prev/next" use-case is a special one (carousel-like)